### PR TITLE
Display last contacted in session

### PR DESCRIPTION
**Pull request by lclarksl**

### CODE REVIEW

File: app/services/CommsApi.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import api from './ApiService.js';
 
-const commsUri = 'http://api.bris.tech:8080';
+const commsUri = 'http://localhost:9001';
```
Comment (pallenSL) [inline app/services/CommsApi.js]: You probably don't want this change in the commit(?)

Comment (jebbinSCL) [inline app/services/CommsApi.js]: Good spot. I guess this should be moved to one of the config files?

**Comment by pallenSL** [comment]
Are lastContact and lastContactDate the same thing?

**Comment by jebbinSCL** [comment]
:shipit: after uri change

File: app/services/CommsApi.js (hunk 1 above)
Comment (pallenSL) [inline app/services/CommsApi.js]: Yes, they would be better in config, however we do that.

**Comment by lclarksl** [comment]
lastContact is a string. lastContactDate is the string wrapped in a momentjs object so that it can be formatted

**Comment by pallenSL** [comment]
Ok.  :boat:

**Pull request merged**